### PR TITLE
feat: prop to enable or disable clipboard

### PR DESCRIPTION
**Pull request by leyenda**
Hello, when using revogrid we use a variety of custom editors, the clipboard function is in conflict with the way we save our data, it could be nice to have a prop to control when the clipboard should be enabled.

Also in this PR I added a condition to only do the custom paste when the editor is closed, which is the normal behavior of sheets or excel.

Have a nice day.

### CODE REVIEW

File: src/components/revo-grid/revo-grid.tsx [hunk 1]
```diff
@@ -59,6 +59,8 @@ export class RevoGridComponent {
   @Prop() resize: boolean = false;
   /** When true cell focus appear. */
   @Prop() canFocus: boolean = true;
+  /** When true enable clipboard. */
+  @Prop() useClipboard: boolean = false;
```
Comment (revolist) [inline src/components/revo-grid/revo-grid.tsx:63]: We need to make it true by default, 'coz other users who used to have it enabled will suffer from it.

**Review by revolist — CHANGES_REQUESTED** [review]
Please change from false to true by default. Thank you for the update.

**Comment by leyenda** [comment]
I am sorry, I was testing and left the value as false by mistake, even in the docs I put the intended value jeje.
Already pushed the change.

**Comment by revolist** [comment]
@leyenda, just fyi docs autogenerated. You don't have to fix them 👍

**Review by revolist — APPROVED** [review]
Thank you for the update. Feel free to contact us for any change.

**Pull request merged**